### PR TITLE
Added Test to User and TodoList

### PR DESCRIPTION
**Pull request by fernandoamz**
Description: I added RSpec to test my Models and fix issues 

What implements/changes?: Just Added Test for Models With RSpec.

How do you manually test this?:  in console run command `rspec`

Before - After screenshots:
captura de pantalla 2018-10-29 a la s 12 50 03

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,18 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```
Comment (8geonirt) [inline db/seeds.rb:8]: Nice :), as a suggestion you can also make use of a cycle to create those objects, just interpolate the index on the name or email or any field you want to make them different 🤓 👍

**Review by 8geonirt — APPROVED** [review]
Nice job 🤓, you can read about `after` and `before` hooks 🤓 , it avoid repeated code on your tests.

File: db/seeds.rb (hunk 1 above)
Comment (fernandoamz) [reply db/seeds.rb:8]: Ok, i will take note about this suggestions and will add in the next commit

File: app/models/todo_list.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 require 'csv'
 
 class TodoList < ApplicationRecord
+  validates :title,:description, presence: true
```
Comment (diegosaul402) [inline app/models/todo_list.rb:4]: you forgot an space before `:description`

File: spec/models/todo_list_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    it 'ensures todo lists is saving' do
+      todo_list_save = TodoList.new(params).save
+      expect(todo_list_save).to eq(true)
+    end
+  end
+
+  context 'review update TodoList' do
+    it 'ensures that the TodoList is Updating content' do
+      todo_list_update = TodoList.find(3).update(params)
+      expect(todo_list_update).to eq(true)
```
Comment (diegosaul402) [inline spec/models/todo_list_spec.rb:38]: I think a better scenario to test is to check if the new values are present and saved after the update

**Review by diegosaul402 — APPROVED** [review]
You are free to attend or not this comments, if not, this PR is ready :shipit:

**Pull request merged**